### PR TITLE
Make auth schemes const

### PR DESCRIPTION
**Pull request by Tratcher**
So you can use them in the Authorize attribute. We already do this for other auth providers.
#5884 HttpSys
#6020 AzureAd

### CODE REVIEW

**Comment by JamesNK** [comment]
Label as a breaking change?

**Comment by Tratcher** [comment]
@JamesNK ack

**Comment by muratg** [comment]
@shirhatti breaking change, so asking for approval from PM side. Thoughts?

**Comment by Tratcher** [comment]
Note these were already approved in security triage.

File: src/Servers/HttpSys/build.cmd [hunk 1]
```diff
@@ -0,0 +1,3 @@
+@ECHO OFF
+SET RepoRoot=%~dp0..\..\..
+%RepoRoot%\build.cmd -projects %~dp0\**\*.*proj %*
```
Comment (muratg) [inline src/Servers/HttpSys/build.cmd]: Is this file change needed, or accidental?

**Review by muratg — APPROVED** [review]
One comment below.

File: src/Servers/HttpSys/build.cmd (hunk 1 above)
Comment (Tratcher) [reply src/Servers/HttpSys/build.cmd]: Added for dev convenience, it has no impact on official builds.

**Pull request merged**